src/js/components/List/stories/action.js

```diff
@@ -12,10 +12,16 @@ const ActionList = () => (
     <Box pad="large">
       <List
         data={data.slice(0, 10)}
-        pad={{ left: 'small', right: 'none' }}
-        action={() => (
-          <Menu icon={<More />} hoverIndicator items={[{ label: 'one' }]} />
-        )}
+        action={(elem, index) => {
```

Comment:
why was the `pad` removed?

Reply:
it created an error message, invalid prop type

Reply:
i can throw it back in

Reply:
ok, back in